pkg/config/application.go

```diff
@@ -554,6 +554,9 @@ func (e *SecretEncryption) Validate() error {
 			return fmt.Errorf("value field of %s in encryptedSecrets must not be empty", k)
 		}
 	}
+	if len(e.DecryptionTargets) == 0 {
+		return fmt.Errorf("no target files to be decrypted")
```

Comment:
[IMO] How about "decryptionTargets must not be empty"?

Reply:
That's because I think users can more easily find the cause&solution if "decryptionTargets" is in the message.